src/afl-fuzz-stats.c

```diff
@@ -375,20 +376,17 @@ void write_stats_file(afl_state_t *afl, u32 t_bytes, double bitmap_cvg,
       "target_mode       : %s%s%s%s%s%s%s%s%s%s\n"
       "command_line      : %s\n",
       (afl->start_time /*- afl->prev_run_time*/) / 1000, cur_time / 1000,
-      runtime / 1000, (u32)getpid(),
+      runtime_ms / 1000, (u32)getpid(),
       afl->queue_cycle ? (afl->queue_cycle - 1) : 0, afl->cycles_wo_finds,
       afl->longest_find_time > cur_time - afl->last_find_time
           ? afl->longest_find_time / 1000
           : ((afl->start_time == 0 || afl->last_find_time == 0)
                  ? 0
                  : (cur_time - afl->last_find_time) / 1000),
-      (runtime -
-       ((afl->calibration_time_us + afl->sync_time_us + afl->trim_time_us) /
-        1000)) /
-          1000,
+      (runtime_ms - MIN(runtime_ms, overhead_ms)) / 1000,
```

Comment:
this is the entry for fuzz_time. runtime_ms - overhead_ms makes sense, sure.
but when would runtime_ms be smaller than overhead_ms?

Reply:
I'm not entirely sure. Initially my working hypothesis was that it was due to the runtime reset on -V (which this commit also removes). However that turned out not to be enough. I didn't dive into this too much, as it seemed to only show up with -V 0 where runtime_ms was off by like single digit values, and on any serious run the problem went away. My best guess is: runtime is tracked in ms, while the other times are tracked in us, leading to slightly different tracking results.

Reply:
yeah the ms/us issue could be it.